### ISSUE TITLE
Sensors Module Settings to hide/show items don't seem to work

### ISSUE THREAD

**Issue by blackketter** [exelban/stats#1317]
**Describe the bug**
I'm trying to slim down the temperatures visible in the Sensors Module.  There are settings for the individual temperatures, but when I turn them off, they stay on in the menu.

In fact, the HID Sensors and Unknown Sensors are the only hide/show controls that work for me.
<img width="1004" alt="Screenshot 2023-02-13 at 8 57 00 AM" src="https://user-images.githubusercontent.com/131679/218521945-f0734aba-2fad-40b4-849d-808a4e0c473b.png">

**Details:**
 - Device: M2 MacBook Air
 - macOS: 13.2
 - Application version: 2.8.8

**Comment by exelban** [comment]
it disables the sensors in the menu bar

**Comment by blackketter** [comment]
Ah, I see.  Sorry about that.  

I was confused by the HID Sensors and Unknown Sensors actually working in the window.  I guess this would be a feature request to adjust what's displayed in the window.

**Comment by exelban** [comment]
yes. because it enables another type of sensor. Please take a look at the opened issues.
